scripts/utilities/clear_generated_images.py

```
@@ -0,0 +1,375 @@
+            
+            confirm = input("\nClear these caption files? (y/N): ").strip().lower()
+            if confirm in ['y', 'yes']:
+                for cf in caption_files:
+                    try:
+                        if cf.is_file():
+                            cf.unlink()
+                            print(f"  ✅ Removed: {cf}")
+                        elif cf.is_dir():
+                            shutil.rmtree(cf)
+                            print(f"  ✅ Removed: {cf}")
+                    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)